src/frontends/paddle/src/op/reverse.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/opsets/opset1.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+
+using namespace default_opset;
+
+NamedOutputs reverse(const NodeContext& node) {
+    auto x = node.get_input("X");
+    // axis is a vector
+    auto axis = node.get_attribute<std::vector<int32_t>>("axis");
+    const auto rank = x.get_partial_shape().rank();
+
+    if (rank == 1) {
```

Comment:
Why do we have to check rank of X here? Cann't we 1:1 map paddle reverse to OV reverse (which has no limitation to rank of X and input axis is 1D instead of scalar)?

Reply:
> Why do we have to check rank of X here? Cann't we 1:1 map paddle reverse to OV reverse (which has no limitation to rank of X and input axis is 1D instead of scalar)?

have fixed it.